src/assets/v1.3.0/compatibility_fallback_handler.json

```diff
@@ -31,7 +31,8 @@
     "80001": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
     "333999": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
     "1313161554": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
-    "1313161555": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4"
+    "1313161555": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
+    "evmos_9000-2": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4"
```

Comment:
Shouldn't this be just `9000`?

Reply:
Yes I'm sorry, I was using the Network Chain ID. It's solved now.

Reply:
No, we will verify the deployment later this week.

Reply:
Hi @rmeissner , I'll have to close this PR due a critical flaw on the testnet. Thank you.
![image](https://user-images.githubusercontent.com/19674406/148561029-f9847a7f-1b5d-48c6-9a35-27fb948ec07c.png)